volttrontesting/platform/auth_test.py

```diff
@@ -187,16 +188,19 @@ def pubsub_authorized(volttron_instance_encrypt, topic='foo', regex=None, peer='
 
 
 @pytest.mark.auth
```

Comment:
All the test_pubsub_unauthorized* tests are marked xfail for now since auth is sending empty capabilities list. @zbeech is looking into the failures.
test_pubsub_authorized is wrongly marked as xfail. I'll remove that.